integration_tests/e2e/about.cy.ts

```diff
@@ -28,4 +31,84 @@ describe('Rendering', () => {
     cy.get('tbody > :nth-child(2) > :nth-child(3)').contains('10 hours')
     cy.get('tbody > :nth-child(2) > :nth-child(4)').contains('3 days')
   })
+
+  it('Should check if main titles of hard-coded high-scoring areas from the assessment are displayed correctly and in order with the correct risk marker', () => {
+    const expectedText = [
+      'Thinking, behaviours and attitudes',
+      'Accommodation',
+      'Personal relationships and community',
+      'Alcohol use',
+      'Employment and education',
+      'Drug use',
+      'Finances',
+      'Health and wellbeing',
+    ]
+    cy.get('.govuk-accordion__section-heading-text-focus')
+      .then($red => $red.toArray().map(el => el.innerText.trim())) // trim whitespace
+      .should('deep.equal', expectedText)
+  })
+
+  it('Should check the hard-coded labels appear next to the correct, predetermined areas in correct order', () => {
+    cy.get('.govuk-accordion__section-summary-focus').eq(1).contains('Risk of reoffending')
+    cy.get('.govuk-accordion__section-summary-focus').eq(3).contains('Risk of reoffending')
+    cy.get('.govuk-accordion__section-summary-focus').eq(4).contains('Risk of reoffending')
+  })
+
+  it('Should check all hard-coded links in each assessment section are in the expected order', () => {
+    cy.get('p.goal-link')
+      .eq(0)
+      .contains('Create thinking, behaviours and attitudes goal')
+      .should('have.attr', 'href')
+      .and('include', '/create-goal/thinking-behaviours-and-attitudes')
+    cy.get('p.goal-link')
+      .eq(1)
+      .contains('Create accommodation goal')
+      .should('have.attr', 'href')
+      .and('include', '/create-goal/accommodation')
+    cy.get('p.goal-link')
+      .eq(2)
+      .contains('Create personal relationships and community goal')
+      .should('have.attr', 'href')
+      .and('include', '/create-goal/personal-relationships-and-community')
+    cy.get('p.goal-link')
+      .eq(3)
+      .contains('Create alcohol use goa')
+      .should('have.attr', 'href')
+      .and('include', '/create-goal/alcohol-use')
+    cy.get('p.goal-link')
+      .eq(4)
+      .contains('Create employment and education goal')
+      .should('have.attr', 'href')
+      .and('include', '/create-goal/employment-and-education')
+    cy.get('p.goal-link')
+      .eq(5)
+      .contains('Create drug use goal')
+      .should('have.attr', 'href')
+      .and('include', '/create-goal/drug-use')
+    cy.get('p.goal-link')
+      .eq(6)
+      .contains('Create finances goal')
+      .should('have.attr', 'href')
+      .and('include', '/create-goal/finances')
+    cy.get('p.goal-link')
+      .eq(7)
+      .contains('Create health and wellbeing goal')
+      .should('have.attr', 'href')
+      .and('include', '/create-goal/health-and-wellbeing')
+  })
+
+  it('Should check if the hard-coded data for high-scoring areas are displayed correctly and in order', () => {
```

Comment:
Test name is a bit ambiguous, and only applies to one of the high scoring sections. More to come?